src/plone/restapi/configure.zcml

```diff
@@ -15,6 +17,15 @@
       provides="Products.GenericSetup.interfaces.EXTENSION"
       />
 
+  <genericsetup:registerProfile
```

Comment:
I prefer to set up testing content types with python code because I think this makes tests far more readable than those obscure zcml profiles. :)

Though, since I don't have much time to help rewriting this code I will just shut up (for now). ;)